audio-worklet/src/bitcrusher-demo.js

```diff
@@ -15,25 +15,70 @@
  */
 class BitcrusherDemo {
   /**
+   * @class BitcrusherDemo
+   * @constructor
    * Initalizes and modifies bitcrusher settings in response to GUI input.
-   * @param  {Number} bitDepth the number of bits for each sample
-   * @param  {Number} reduction the amount of sample rate reduction to apply
-   * @param  {Number} gain the volume of the output
+   * @param {AudioContext} context the Audio Context
+   * @param {Boolean} enableWorklet switch depending on browser
+   *                  which determines if both script processor and audio
+   *                  worklet should be used. If false, only the script
+   *                  processor will be used.
    */
-  constructor(containerId, bitDepth, reduction, gain) {
-    this.context_ = new AudioContext();
-    this.masterGain_ = new GainNode(this.context_, {gain: (gain || 0.5)});
-    this.bitcrusher_ = new Bitcrusher(this.context_, {
-      channelCount: 1,
-      bitDepth: bitDepth || 24,
-      reduction: reduction || 1
-    });
+  constructor(context,  enableWorklet) {
+    this.context_ = context;
+    this.masterGain_ = new GainNode(this.context_, {gain: 0.5});
+    this.enableWorklet_ = enableWorklet;
```

Comment:
Haha. I thought the original `useWorklet` and suggested `enableWorklet` as a better name.